src/model.ts

```diff
@@ -128,7 +133,7 @@ export class JSORMBase {
   static endpoint: string
   static isBaseClass: boolean
   static jwt?: string
-  static camelizeKeys: boolean = true
+  static keyCase: KeyCase = { from: "snake", to: "camel" }
```

Comment:
`from` and `to` are confusing names for me when the possible arguments are the case themselves.  Transitive names of these imply that the arguments are functions, when instead they are 'which case I am in when on a particular side of the server and client interaction'.  For this reason I'd suggest renaming to `client`  and either `server` or `api`.